romulus.xml

```diff
@@ -44204,6 +44208,10 @@
 		<id>TYPE</id>
 		<default>NA</default>
 	</attribute>

```

Comment:
This looks like a closing XML tag without an opening tag.

example from Witherspoon.
<targetInstance>
	<id>module-0</id>
	<type>module-module-monza</type>
	<is_root>false</is_root>
	<instance_name>module</instance_name>
	<position>0</position>
	<child_id>p9_proc_m</child_id>
	<attribute>
		<id>AFFINITY_PATH</id>
		<default></default>
	</attribute>